.travis.yml

```diff
@@ -1,20 +1,26 @@
 ---
 dist: bionic
```

Comment:
Should we use focal instead (ubuntu 20)?

Reply:
done